cmake/Platform/Arduino.cmake

```diff
@@ -2269,6 +2307,7 @@ function(get_arduino_flags COMPILE_FLAGS_VAR LINK_FLAGS_VAR BOARD_ID MANUAL)
             else ()
                 set(ARDUINO_VERSION_DEFINE "${ARDUINO_VERSION_DEFINE}0${CMAKE_MATCH_2}")
             endif ()
+            set(ARDUINO_VERSION_DEFINE "${ARDUINO_VERSION_DEFINE}00")
```

Comment:
I don't quite understand what does it change, can you explain it some more? @noseglasses